frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -599,7 +599,7 @@ describe("scenarios > dashboard > dashboard drill", () => {
 
   it('should drill-through on PK/FK to the "object detail" when filtered by explicit joined column (metabase#15331)', () => {
     cy.server();
-    cy.route("POST", "/api/dataset").as("dataset");
+    cy.route("POST", "/api/card/*/query").as("cardQuery");
```

Comment:
Not using `/api/dataset` when browsing object details anymore because the filters approach is replaced 🎉

Reply:
@kulyk Note for the future - when you encounter `cy.server()` and `cy.route()` and you need to change something, please also change it to `cy.intercept()`.